app/controllers/application_controller.rb

```diff
@@ -129,4 +129,14 @@ def suggest_email_notifications?
 
       $redis.sismember("email_notifications_reminder", current_user.id.to_s)
     end
+
+    def load_participation_process
+      if params[:participatory_process_id].present?
+        @participatory_process_id = params[:participatory_process_id]
+        @participatory_process = ParticipatoryProcess.find(params[:participatory_process_id])
+      else
+        default_participatory_process_id = ParticipatoryProcess.first.slug
```

Comment:
I would only resort to this when on the index action, in order to redirect. It's dangerous to just assume it's gonna be the first one in the rest of scenarios.